Formula/micro.rb

```diff
@@ -0,0 +1,97 @@
+require "language/go"
+
+class Micro < Formula
+  desc "Modern and intuitive terminal-based text editor"
+  homepage "https://github.com/zyedidia/micro"
+  url "https://github.com/zyedidia/micro/archive/v1.0.3.tar.gz"
+  sha256 "dc9a693d9eedbbf5a0048a092172a56de6b2b608f7d1419274c12416a840876d"
+  head "https://github.com/zyedidia/micro.git"
+
+  depends_on "go" => :build
+
+  go_resource "github.com/gdamore/encoding" do
+    url "https://github.com/gdamore/encoding.git",
+    :revision => "b23993cbb6353f0e6aa98d0ee318a34728f628b9"
```

Comment:
The current agreement among maintainers is that `:revision` should be aligned with the opening quote, i.e.,

```ruby
   go_resource "github.com/gdamore/encoding" do
     url "https://github.com/gdamore/encoding.git",
         :revision => "b23993cbb6353f0e6aa98d0ee318a34728f628b9"
```